### PR TITLE
Email verification institute emails

### PR DESCRIPTION
**Pull request by Thomass1412**
### Description
Now when you signup with an institute email eg. @student.aau.dk an email will be sent to the user with a 4 digit code. When it has been sent a pop-up will appear where the user can verify the code. If the code is correct the user will be directed to the main page, logged in
### Changes
added new component emailverification, with 3 files, One for the overall modal where props are passed from. One for the footer of the component, handles continue and resend email buttons. One for the pinfield. The component is loaded from the signup page. 
### Related Issues
https://github.com/orgs/Educado-App/projects/14/views/12?pane=issue&itemId=78205350

image

### CODE REVIEW

File: app/src/components/emailVerification/CodeVerification.tsx
Comment (MartinJenzen) [inline app/src/components/emailVerification/CodeVerification.tsx:1]: Mucho bueno 👍 Men overvej evt. at have en separat **.css** til styles. Se evt. **GenericModalComponent.css** i **app/src/components/styles**.

File: app/src/components/emailVerification/EmailVerificationModal.tsx
Comment (MartinJenzen) [inline app/src/components/emailVerification/EmailVerificationModal.tsx:1]: Would be nice with a little quick comment above line 76, e.g.: {/* Reset password  */}

File: app/src/components/emailVerification/NavigationFooter.tsx
Comment (MartinJenzen) [inline app/src/components/emailVerification/NavigationFooter.tsx:1]: Rename "onSubmit1" variable back to "onSubmit". And _maybe_ add a small descriptive comment above line 103 and 114/115.

**Review by MartinJenzen — APPROVED** [review]
Borat: "_Very nice!_". But look at comments and consider adjusting those minor details.

File: app/src/components/emailVerification/CodeVerification.tsx
Comment (Todua992) [reply app/src/components/emailVerification/CodeVerification.tsx:1]: Fordi det er Inline Tailwind css så kan vi ikke som sådan lave et fælles eller separat css filen.

File: app/src/components/emailVerification/EmailVerificationModal.tsx
Comment (Todua992) [reply app/src/components/emailVerification/EmailVerificationModal.tsx:1]: Nu hvor du siger det, Så opdagede vi at vi har skrevet  Reset Password som titlen, I stedet skulle der stå Verificerer Email, Men vi tilføjer også komenter :)

File: app/src/components/emailVerification/CodeVerification.tsx
Comment (MartinJenzen) [reply app/src/components/emailVerification/CodeVerification.tsx:1]: Det kan man faktisk godt, se hvad jeg har gjort i **GenericModalComponent.tsx** og i **GenericModalComponent.css**. Spørgsmålet er dog, om det er dét værd at lave for kun 8 linjer. Det var også bare en suggestion ;)

**Pull request merged**